### PR TITLE
feat: notifications datatestid

### PR DESCRIPTION
**Pull request by rish2320**
# Description

Added the data-testids for the notification automation

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/app/details/appDetails/AppDetails.tsx [hunk 1]
```diff
@@ -508,7 +508,7 @@ export const Details: React.FC<DetailsType> = ({
             )}
             {loadingResourceTree ? (
                 <div className="bcn-0 dc__border-top h-100">
-                    <Progressing pageLoader fullHeight size={32} fillColor="var(--N500)" />
```
Comment (vivek-devtron) [inline src/components/app/details/appDetails/AppDetails.tsx:511]: is it expected?

File: src/components/notifications/ConfigurationTab.tsx [hunk 2]
```diff
@@ -101,7 +101,7 @@ export class ConfigurationTab extends Component<{}, ConfigurationTabState> {
         return (
             <div key="slack-config" className="white-card white-card--configuration-tab mb-16">
                 <div className="configuration-tab__header">
-                    <p className="configuration-tab__title">
+                    <p data-testid = "slack-heading-title" className="configuration-tab__title">
```
Comment (vivek-devtron) [inline src/components/notifications/ConfigurationTab.tsx:104]: format this line

File: src/components/notifications/ConfigurationTab.tsx [hunk 3]
```diff
@@ -231,7 +235,7 @@ export class ConfigurationTab extends Component<{}, ConfigurationTabState> {
         return (
             <div key="smtp-config" className="white-card white-card--configuration-tab">
                 <div className="configuration-tab__header">
-                    <p className="configuration-tab__title flexbox">
+                    <p data-testid = "smtp-heading-title" className="configuration-tab__title flexbox">
```
Comment (vivek-devtron) [inline src/components/notifications/ConfigurationTab.tsx:238]: same here

File: src/components/notifications/SESConfigModal.tsx [hunk 4]
```diff
@@ -325,6 +328,7 @@ export class SESConfigModal extends Component<SESConfigModalProps, SESConfigModa
                                 AWS Region*
                             </label>
                             <ReactSelect
+                                classNamePrefix='add-ses-aws-region'
```
Comment (vivek-devtron) [inline src/components/notifications/SESConfigModal.tsx:331]: format this line

File: src/components/app/details/appDetails/AppDetails.tsx (hunk 1 above)
Comment (rish2320) [reply src/components/app/details/appDetails/AppDetails.tsx]: done

File: src/components/notifications/ConfigurationTab.tsx [hunk 5]
```diff
@@ -207,7 +210,7 @@ export class ConfigurationTab extends Component<{}, ConfigurationTabState> {
         return (
             <div key="ses-config" className="white-card white-card--configuration-tab">
                 <div className="configuration-tab__header">
-                    <p className="configuration-tab__title">
+                    <p data-testid = "ses-heading-title" className="configuration-tab__title">
```
[the remark this answers is not in the document]
Comment (rish2320) [reply src/components/notifications/ConfigurationTab.tsx]: done

File: src/components/notifications/ConfigurationTab.tsx (hunk 3 above)
Comment (rish2320) [reply src/components/notifications/ConfigurationTab.tsx]: done

File: src/components/notifications/SESConfigModal.tsx (hunk 4 above)
Comment (rish2320) [reply src/components/notifications/SESConfigModal.tsx]: done

File: src/components/v2/appDetails/AppDetails.component.tsx [hunk 6]
```diff
@@ -75,7 +75,7 @@ const AppDetailsComponent = ({
             )}
             {loadingResourceTree ? (
                 <div className="bcn-0 dc__border-top h-100">
-                    <Progressing pageLoader fullHeight size={32} fillColor="var(--N500)" />
```
[the remark this answers is not in the document]
Comment (rish2320) [reply src/components/v2/appDetails/AppDetails.component.tsx]: done

File: src/components/notifications/AddNotification.tsx [hunk 7]
```diff
@@ -735,6 +741,7 @@ export class AddNotification extends Component<AddNotificationsProps, AddNotific
             <div className="form__row">
                 <p className="add-notification__title mb-16">Send to</p>
                 <CreatableSelect
+                    classNamePrefix='add-notification-send-to'
```
Comment (vivek-devtron) [inline src/components/notifications/AddNotification.tsx:744]: format this line

**Review by vivek-devtron — APPROVED** [review]

File: src/components/notifications/AddNotification.tsx (hunk 7 above)
Comment (rish2320) [reply src/components/notifications/AddNotification.tsx]: done

**Pull request merged**